instrumentation/kamon-finagle/src/main/scala/kamon/instrumentation/finagle/client/Tags.scala

```diff
@@ -0,0 +1,63 @@
+package kamon.instrumentation.finagle.client
+
+import com.twitter.finagle.http.Status
+import com.twitter.util.Time
+import kamon.trace.Span
+
+import java.net.InetSocketAddress
+
+/**
+ * Unique tags and value constants added to Kamon Finagle spans.
+ *
+ * As a general rule metric tags should only be for tags with low cardinality.
+ */
+private[finagle] object Tags {
+
+  object Keys {
```

Comment:
There are a few tags that I don't recall using in the past, could you please share what made you include them? I'm talking about:
  - `span.type`
  - `error.type`
  - `http.status_category`

Reply:
I addressed `span.type` in the other comment.

`error.type` this is something we use internally, but now that I think about it it's something we can instead define as a `mark` when errors occur for certain finagle operation events, so I'll make that change and drop the tag.

`http.status_category` another internal concern I'll remove from this implementation.

Reply:
Great! please ping me when you add those changes to look at this again.